App.js

```diff
@@ -303,7 +310,7 @@ export default class App extends React.Component {
         <StatusBar backgroundColor={this.styles.statusBar.backgroundColor} barStyle="default" />
         <WebView
           domStorageEnabled
-          injectedJavaScript={this.appInfoJavaScript}
+          injectedJavaScript={this.injectedJavaScript}
```

Comment:
I'm not sure this line is actually needed at all, since we're loading the same code on every `onLoadEnd`, but I donno :woman_shrugging: